test/liquid_voting_web/absinthe/mutations/create_delegation_test.exs

```diff
@@ -190,6 +190,29 @@ defmodule LiquidVotingWeb.Absinthe.Mutations.CreateDelegationTest do
       assert message == "Could not create delegation"
       assert details == %{delegate_id: ["can't be blank"]}
     end
+
+    test "with identical emails for delegator and delegate" do
```

Comment:
We may need a test for creating a delegation with IDs as well?

Reply:
Good point. I'll add this.

Reply:
done